backend/src/main/scala/cromwell/backend/Backend.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.backend
+
+import cromwell.backend.model.{WorkflowDescriptor, JobDescriptor}
+
+/**
+  * Defines basic structure and functionality to work with a backend.
+  */
+trait Backend {
```

Comment:
It seems there's parallel but distinct nomenclature on the engine and backend sides for workflow-level versus call-level.   e.g.

WorkflowActor -> BackendActor
CallActor -> BackendJobExecutorActor

Could we try to make this more consistent to reduce confusion?